components/Addressing/models.rst

```diff
@@ -1,262 +1,222 @@
++-------------+------------------------------------+
+| city        | Address' city                      |
++-------------+------------------------------------+
+| postcode    | Address' postcode                  |
++-------------+------------------------------------+
+| createdAt   | Date when address was created      |
++-------------+------------------------------------+
+| updatedAt   | Date of last address' update       |
++-------------+------------------------------------+
+
+
+.. note::
+   This model implements the :ref:`component_addressing_model_address-interface`. |br|
+   For more detailed information go to `Sylius API Address`_.
+
+.. _Sylius API Address: http://api.sylius.org/Sylius/Component/Addressing/Model/Address.html
+
+.. _component_addressing_model_country:
 
 Country
 -------
 
-"Country" model represents a geographical area of a country.
-
-+-----------+--------------------------------+
-| Attribute | Description                    |
-+===========+================================+
-| id        | Unique id of the country       |
-+-----------+--------------------------------+
-| name      |                                |
-+-----------+--------------------------------+
-| isoName   |                                |
-+-----------+--------------------------------+
-| provinces | Collection of provinces        |
-+-----------+--------------------------------+
-| createdAt | Date when country was created  |
-+-----------+--------------------------------+
-| updatedAt | Date of last country update    |
-+-----------+--------------------------------+
-
-.. code-block:: php
-
-    <?php
-
-    use Sylius\Component\Addressing\Address;
-    use Sylius\Component\Addressing\Country;
-    
-    $poland = new Country();
-    $poland
-        ->setName('Poland')
-        ->setIsoName('PL')
-    ;
-
-    $address
-        ->setFirstname('Pawel')
-        ->setLastname('Jedrzejewski')
-        ->setCountry($poland)
-        ->setStreet('Testing 123')
-        ->setCity('Lodz')
-        ->setPostcode('21-242')
-    ;
+The geographical area of a country is represented by a **Country** instance.
+It should contain all data concerning a country and as default has the following properties:
+
++-----------+--------------------------------------+
+| Property  | Description                          |
++===========+======================================+
+| id        | Unique id of the country             |
++-----------+--------------------------------------+
+| isoName   | Country's ISO code                   |
++-----------+--------------------------------------+
+| provinces | Collection of **Province** objects   |
++-----------+--------------------------------------+
+| enabled   | Indicates whether country is enabled |
++-----------+--------------------------------------+
+| createdAt | Date when the object was created     |
++-----------+--------------------------------------+
+| updatedAt | Date of last object's update         |
```

Comment:
Actually.. it does not implement `TimestampableInterface`, so it does not have createdAt and updatedAt fields.